tests/unit/services/metadata-test.js

```diff
@@ -1,12 +0,0 @@
-import { module, test } from 'qunit';
-import { setupTest } from 'ember-qunit';
-
-module('Unit | Service | metadata', function(hooks) {
-  setupTest(hooks);
-
-  // Replace this with your real tests.
-  test('it exists', function(assert) {
```

Comment:
One thing I was thinking, maybe it would be better to mark these tests as `pending` for now instead of deleting them? 
https://coderwall.com/p/3l8daw/mark-ember-js-spec-as-pending

Reply:
Reverted deletes and addressed this in 31c9a78

Reply:
Thanks!